packages/compass-components/src/components/breadcrumb.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { spacing } from '@leafygreen-ui/tokens';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { palette } from '@leafygreen-ui/palette';
+import { useDarkMode } from '../hooks/use-theme';
+import { Link, Icon, Body } from './leafygreen';
+
+export type BreadcrumbItem = {
+  name: string;
+  onClick: () => void;
+};
+
+const breadcrumbStyles = css({
+  display: 'flex',
+  gap: spacing[1],
+  alignItems: 'center',
+  height: spacing[4],
+});
+
+const dbLinkLightStyles = css({
+  color: palette.green.dark2,
+});
+
+const dbLinkDarkStyles = css({
+  color: palette.green.base,
+});
+
+const textStyles = css({
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap',
+  fontWeight: 'bold',
+});
+
+const lastItemStylesLight = css({
+  color: palette.gray.dark1,
+});
+
+const lastItemStylesDark = css({
+  color: palette.gray.base,
+});
+
+export const Breadcrumbs = ({ items }: { items: Array<BreadcrumbItem> }) => {
+  const darkMode = useDarkMode();
+  return (
+    <div className={breadcrumbStyles} data-testid="breadcrumbs">
+      {items.map((item, index) => {
+        const isLast = index === items.length - 1;
+        if (isLast) {
+          return (
+            <Body
+              className={cx(
+                textStyles,
+                darkMode ? lastItemStylesDark : lastItemStylesLight
+              )}
+            >
+              {item.name}
+            </Body>
+          );
+        }
+        return (
+          <>
+            <Link
+              key={item.name}
+              as="a"
```

Comment:
Because we're not handling this as a link and there's no url, this should be as button, right?

```suggestion
              as="button"
```